<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSingleSelectTimeWidget</class>
 <widget class="QWidget" name="QSingleSelectTimeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSingleSelectTimeWidget</string>
  </property>
  <widget class="QPushButton" name="btnUp">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>向上</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDown">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>向下</string>
   </property>
  </widget>
  <widget class="QLabel" name="labPrevious">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>上一个时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="labCurrent">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>当前时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="labNext">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>下一个时间</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
